examples/serve/http_server/task.yaml

```diff
@@ -10,7 +10,8 @@ service:
   readiness_probe:
     path: /health
     initial_delay_seconds: 20
-  replicas: 2
+  replica_policy:
+    min_replicas: 2
```

Comment:
Do we want to still use `replicas: 2` for simplicity?

Reply:
oops good catch